### PR TITLE
Changelog for 1.3.0

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## 1.2.2 (06/04/2018)
+
+### Changes
+
+ * Added SystemV shared memory based CacheItemPool (#191)
+   Fixes usage of deprecated env var for GAE Flex #189
```
Comment (bshaffer) [inline CHANGELOG.md]: The formatting is off here.  should be

```
 * Added SystemV shared memory based CacheItemPool (#191)
 * Fixes usage of deprecated env var for GAE Flex (#189)
```

**Comment by bshaffer** [comment]
This should be `1.3.0` because we are adding a feature (sysv)

**Comment by bshaffer** [comment]
Also include notes on the fix in https://github.com/google/google-auth-library-php/pull/190

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+## 1.3.0 (06/04/2018)
+
+### Changes
+
+ * Fixes usage of deprecated env var for GAE Flex #189
+ * fix - guzzlehttp/psr7 dependency version definition #190
```
[the remark this answers is not in the document]
Comment (tmatsuo) [reply CHANGELOG.md]: done

**Pull request merged**